packages/breadcrumb/src/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { ChevronRight } from '@project44-manifest/react-icons';
+import { cx } from '@project44-manifest/react-styles';
+import type { ForwardRefComponent } from '@project44-manifest/react-types';
+import { StyledBreadcrumbItem } from './BreadcrumbItem.styles';
+import type { BreadcrumbItemElement, BreadcrumbItemProps } from './BreadcrumbItem.types';
+
+export const BreadcrumbItem = React.forwardRef((props, forwardedRef) => {
+  const { as, children, className: classNameProp, css, ...other } = props;
+
+  const className = cx('manifest-breadcrumb-item', classNameProp);
+
+  return (
+    <StyledBreadcrumbItem {...other} ref={forwardedRef} as={as} className={className} css={css}>
+      {children}
```

Comment:
I think we would use our `Link` component here to ensure visual consistency and ease of markup. The BreadcrumbItem could take in an `href` as a prop or just extend the `Link` prop. 

I guess this would require splitting out the link into its own component as well.